radarpipeline/io/reader.py

```diff
@@ -202,9 +110,9 @@ def read(self) -> RadarData:
                 if variable_data.get_data_size() > 0:
                     variable_data_dict[dirname] = variable_data
 
-            user_data_dict[uid] = RadarUserData(variable_data_dict)
+            user_data_dict[uid] = RadarUserData(variable_data_dict, self.df_type)
```

Comment:
what is the `self.df_type` is doing here?

Reply:
I had refactored all the datalib modules to use both pyspark and pandas DataFrames. This parameter is used so that in case any calculation requires a implementation specific operation then we can do them conditional on it.

Reply:
Ah makes sense. Thanks